ASP/HTML-ASP.sublime-syntax

```diff
@@ -14,9 +14,10 @@ contexts:
         - match: "%>"
           scope: punctuation.section.embedded.end.asp
           pop: true
-        - match: (').*?(?=%>)
-          scope: comment.line.apostrophe.asp
-          captures:
-            1: punctuation.definition.comment.asp
-        - include: scope:source.asp
+        - match: ''
+          push:
+            - include: scope:source.asp # don't directly push, so that the scope doesn't also contain 'source.asp ' as well as our 'source.asp.embedded.html '
+          with_prototype:
+            - match: "(?=%>)"
+              pop: true
```

Comment:
@michaelblyons, I imagine it should work a bit like this, in case it helps to clarify:

    <div> <!-- root html context `text.html.basic` -->
    <% // root level <%, so the scope would be something like `text.html.basic source.cs.embedded`
       if (condition) { %> <!-- as the %> is inside the "if block" context, it should push a new HTML context onto the stack -->
          <p>Hello World</p><!-- so the scope would be something like `text.html.basic source.cs.embedded meta.block.source.cs text.html.basic`
        <% // this should pop the html context, and we are back to `text.html.basic source.cs.embedded meta.block.source.cs`
       } // this will pop the `meta.block.source.cs`
     %> <!-- this will pop the root `source.cs.embedded` -->
    </div> <!-- so we are back to `text.html.basic` -->